Your Portfolio/Chickisnak.html

```diff
@@ -505,7 +505,7 @@ <h1 class="m-0 display-5"><span class="text-primary">Akhand </span>Pratap Singh<
                     <!-- TRY TO KEEP YOUR IMAGE IN 1:1,THAT IS, SQUARE -->
 
                     <img class="img-fluid w-100 rounded-circle shadow-sm" 
-                    src="210111005.jpeg"
+                    src="Images/Chickisnak.jpeg"
```

Comment:
Can you make this change in the template as well?

Reply:
I've made the requested changes, please check them out